experiments/ly5320.py

```diff
@@ -21,11 +21,13 @@
 from xcs.db import daq
```

Comment:
Is ly5320 a standard one that they'll be copying from/using in the future?

Reply:
This is something I could use some guidance on.  I believe it's the experiment that inspired the JIRA ticket, and the most recent experiment when I updated the repo.  

There doesn't seem to be a template experiment file, to the best of my knowledge.

Reply:
Having a template for new experiments would be useful. 
That said, we will be copying the current experiment file to setup the next experiment since it is a continuation of a series of liquid jet experiments with the same setup.

Reply:
I think the better long term solution might be to move some of the repeated experiment functionality into the xcs "library" or `beamline.py` file.  Things like scan definitions are being repeated from experiment to experiment with very minimal changes.